uvtools/dspec.py

```diff
@@ -12,6 +12,13 @@
 from scipy.optimize import leastsq, lsq_linear
 import copy
 
+#DEFAULT PARAMETERS FOR CLEANs
```

Comment:
can you add a brief comment here that "True" is for 2D filtering and "False" is for 1D filtering

Reply:
I've added a comment explaining the switches.